components/gc-minister/gc-minister-en.html

```diff
@@ -0,0 +1,37 @@
+---
+{
+	"title": "Minister or institutional head",
+	"description": "Working example for the 'Minister or institutional head' component",
+	"language": "en",
+	"altLangPage": "gc-minister-fr.html",
+	"breadcrumbs": [
+		{ "title": "Minister or institutional head - Documentation", "link": "components/gc-minister/gc-minister-doc-en.html" }
+	],
+	"dateModified": "2024-03-11",
+}
+---
+<div class="wb-prettify all-pre hide"></div>
+
+<p>Below are working examples for the "Minster or institutional head" component.</p>
+
+<h2>Example with only one minister or institutional head</h2>
+{% include_relative samples/gc-minister.html %}
```

Comment:
Having consecutive heading at the same level where they do related hierarchically are problematic.

The easier will be to have one example per page because the text of the h2 is static with 2 alternative (plural or singular form)

Or, if you want to keep it on the same page, I suggest the following correction:
* Add a little note, in a panel directly under the h1, regarding the ministerial heading with some specificity of its context of use and that we removed its semantic bellow only for the example purpose
* Add an include parameter that allow to deactivate the heading level 2, like rendering it as paragraph.